### ISSUE TITLE
encyclopedia entry not working

### ISSUE THREAD

**Issue by callmepartario** [heclak/community-a4e-c#509]
Our encylopedia entry appears to have gone defunct. I'll take a look at the F-14 as a template for a new entry into this oh-so-vital DCS function, available from the main menu, which, of course, every single module supports in ubiquity.

**Software Information (please complete the following information):**
 - DCS Version: [e.g. DCS 2.7.10.18996 Open Beta]
 - A-4E Version [e.g. v2.0.0 dev]

**Comment by nima3333** [comment]
Looks like it appears again when you modify the name of the entry to something else (first line of the text file), strange

**Comment by callmepartario** [comment]
yeah, i think DCS is looking for mods and their encyclopedia entries. i've yet to find a module with a fully working one. i've made an update in https://github.com/heclak/community-a4e-c/commit/4601d338a4d58b9bf5280a3e1a410c94636bf4ac so as to create a duplicate entry so it's at least there, but this is less than ideal.

**Comment by callmepartario** [comment]
`A-4E-C. lua` -- add `encyclopedia_hidden = true`

`Encyclopedia/Plane/A-4E-C.txt` add "-C"
